proposals/0296-increase-tx-size.md

```diff
@@ -122,10 +104,12 @@ not be included in the chain. Violations are considered sanitization failures:
 | max num signatures per transaction | 42 | new |
 | max num accounts | 96 | new |
 | max num instructions | 64 | old |
-| max accounts/instruction | 255 | new |
+| max accounts/instruction | 255 | old |
```

Comment:
Sorry, I just checked and this is actually new. We currently only have a _runtime_ limit of 255 accounts/ix for top-level bpf program instructions and for any CPI instruction. There's no protocol constraint on accounts/ix so it's currently valid to pass over 255 accounts to a top-level builtin program instruction so this would be a new protocol constraint.

Reply:
Updated back